src/extension/ios/plistBuddy.ts

```diff
@@ -128,6 +130,11 @@ export class PlistBuddy {
     }
 
     public getInferredScheme(iosProjectRoot: string, projectRoot: string, rnVersion: string) {
+        const projectWorkspaceConfigName = this.getProjectWorkspaceConfigName(iosProjectRoot, projectRoot, rnVersion);
+        return getFileNameWithoutExtension(projectWorkspaceConfigName);
+    }
+
+    public getProjectWorkspaceConfigName(iosProjectRoot: string, projectRoot: string, rnVersion: string): string {
```

Comment:
Not sure if we need to rename this function since it was added from React Native CLI to get default RN scheme after 0.59 version.